core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java

```diff
@@ -218,7 +218,9 @@ static String resolveDockerHostIpAddress(DockerClient client, URI dockerHost) {
                         .filter(it -> it.getGateway() != null)
                         .findAny()
                         .map(Network.Ipam.Config::getGateway)
-                        .orElse("localhost");
+                        .orElseGet(() -> {
+                            return DockerClientConfigUtils.getDefaultGateway().orElse("localhost");
+                        });
```

Comment:
```suggestion
                        .orElseGet(() -> DockerClientConfigUtils.getDefaultGateway().orElse("localhost"));
```

Reply:
No, thank you, it is done for readability :)

Reply:
ok, thought you had multiline lambda to begin with, but ended up in oneliner later on forgetting to remove the braces :)

Reply:
ah, no, just I don't like "long short lambdas" (lol). Thanks for suggesting anyways! 👍

Reply:
> long short lambdas
🤣